### PR TITLE
Griefergames

### PR DESCRIPTION
**Pull request by FlexKleks**
## Type of change

- [ x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [ x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/griefergames/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "server_name": "GrieferGames",
+    "nice_name": "GrieferGames Netzwerk",
+    "direct_ip": "griefergames.net",
+    "social": {
+      "web": "https://griefergames.net/",
+      "web_shop": "https://store.griefergames.net/",
+      "web_support": "https://bit.ly/griefergames-support",
```
Comment (JakobBruening) [inline minecraft_servers/griefergames/manifest.json:8]: Please use the direct url, not https://bit.ly/.

File: minecraft_servers/griefergames/manifest.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "server_name": "GrieferGames",
```
Comment (JakobBruening) [inline minecraft_servers/griefergames/manifest.json:2]: As it is already requested, the `server_name` has to be the directory name.

File: minecraft_servers/griefergames/manifest.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "nice_name": "GrieferGames Netzwerk",
+    "direct_ip": "griefergames.net",
+    "social": {
+      "web": "https://griefergames.net/",
+      "web_shop": "https://store.griefergames.net/",
+      "web_support": "https://bit.ly/griefergames-support",
+      "twitter": "GrieferGamesNW",
+      "facebook": "GrieferGamesNW",
+      "instagram": "griefergamesnw",
+      "youtube": "https://www.youtube.com/user/AbgegrieftHD",
```
Comment (JakobBruening) [inline minecraft_servers/griefergames/manifest.json:12]: Since it's not the network's YouTube account as such, I would remove it.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
- Please compress the images
- Please use an icon with better quality
- Feel free to upload a logo.png, I saw a good image on forum.griefergames.net

File: minecraft_servers/griefergames/manifest.json (hunk 3 above)
Comment (FlexKleks) [reply minecraft_servers/griefergames/manifest.json]: I removed it

File: minecraft_servers/griefergames/manifest.json (hunk 2 above)
Comment (FlexKleks) [reply minecraft_servers/griefergames/manifest.json]: Yeah, i changed the server name

File: minecraft_servers/griefergames/manifest.json (hunk 1 above)
Comment (FlexKleks) [reply minecraft_servers/griefergames/manifest.json]: URL is changed to the forum

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Looks better now, thank you! But there are still a few issues:

- The logo.png looks a little bit streched :) - And it would be great adding `logo@2x.png`, if `logo.png` is added
- `icon.png` and `icon@2x.png` are required - You have to add the files, even if the logo-files are added

**Comment by FlexKleks** [comment]
Unfortunately, the Logo.png cannot be reduced any further. Since this otherwise looks very pixelated.

**Review by JakobBruening — APPROVED** [review]
Looks great now, thank you!

**Pull request merged**